### DISCUSSION TITLE
Levelbody invertiert nutzen nicht von 0->100 sondern von 100->0 funktioniert nicht

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1888] [Q&A]
`v3.1.0-alpha.46`
![image](https://user-images.githubusercontent.com/84818934/191370269-af7cc6a2-fd54-425e-a744-0c27c8f5518b.png)
![image](https://user-images.githubusercontent.com/84818934/191370325-0c41892e-952a-406e-a7aa-d1cfd5b068d1.png)

Datenpunkt-Eigenschaften
`{"value":"val=> (100-val)"}` oder `{"value":"val => Math.floor(100-val)"}`
`"min":100,"max":0` ergänzen hilft auch nicht.

Marker
`(val=>100-val)`
![image](https://user-images.githubusercontent.com/84818934/191370496-9b42b43c-1bc8-4ca0-990a-7006844ea9d8.png)

LevelBody liefert immer den Standard LevelBodywert an der Auswahlstelle.

**Comment by Zefau** [comment]
Jo, die Einstellungsmöglichkeiten sind alle lediglich für die Anzeige und rein visuell. `setOn` und `setOff` sind zum Schalten, unterstützen aber keinen Callback.

**Reply by mcuiobroker** [reply]
Kannst du was machen? Feature Request?
